lib/manager/terraform/readme.md

```diff
@@ -1,19 +1,36 @@
-Currently Terraform support is limited to Terraform registry sources and github sources that include SemVer refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
+Currently, Terraform supports renovating the following dependencies, where sub points represent hosting options of the dependencies:
```

Comment:
```suggestion
Terraform supports renovating the following dependencies, where sub points represent hosting options of the dependencies:
```

To me the use of the words "Currently, " implies that on a later date Renovate/Terraform will support more renovations. By not using such words as "currently, later, right now, in future" we don't create expectations of further improvements. The sentence would read just as nice if we removed the word "Currently, ".

Also by not using words like "Currently" we free ourselves from updating the docs when at a later time "Currently, Terraform supports..." some other renovation as well.

So overall, I think using "time-based" language like this is a bit dangerous.

But let's see what the rest of the Renovate team thinks about using words like this in our docs... 
So don't rush into applying this suggestion right away, as I'm curious what the rest of the Renovate team has to say. Maybe I'm totally honking up the wrong pond here... 😄